Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>Microsoft.Toolkit.Uwp.SampleApp</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
```

Comment:
@azchohfi just had to make this one small fix for the WinUI update, sample app seemed to run fine otherwise.

Did it work alright on a device?

Reply:
It did. These seems to be an issue with the Sample app because of WinUI, but the generated nugets works fine. Let's merge this and investigate the Sample app issue later.